libsplinter/src/service/scabbard/client/key.rs

```diff
@@ -0,0 +1,93 @@
+// Copyright 2020 Cargill Incorporated
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::fs::File;
+use std::io::{BufRead, BufReader};
+
+use sawtooth_sdk::signing::secp256k1::Secp256k1PrivateKey;
```

Comment:
The scabbard client should probably support the generic signing API in libsplinter itself rather than relying explicitly on the sawtooth SDK.